### PR TITLE
Implement Required Static Criteria and Validation of Static Criteria within the Employee Portfolio Components

### PR DESCRIPTION
**Pull request by molliemorrow**
Completed implementation for static criteria and validation of static criteria for AboutMe, IndustryEquivalancey, Education, Work Experience, Other Work Experience, Projects, Certifications, Honors&Awards, and phase submission.

### CODE REVIEW

File: devops/azure-pipelines/azure-pipelines.yml
Comment (ddinh) [inline devops/azure-pipelines/azure-pipelines.yml]: File still contains merge conflicts

File: devops/azure-pipelines/build-job-template.yml
Comment (ddinh) [inline devops/azure-pipelines/build-job-template.yml]: File still contains merge conflicts

File: devops/azure-pipelines/discord-job-template.yml
Comment (ddinh) [inline devops/azure-pipelines/discord-job-template.yml]: File still contains merge conflicts

**Review by MadeFaux — DISMISSED** [review]
Please try and pull from the main branch more often to avoid all of these merge errors :)

**Review by sophiagavrila — CHANGES_REQUESTED** [review]
@molliemorrow just need a few changes to be made that are being picked up by SonarCloud as bugs.

1.  Go to[ this SonarCloud](https://sonarcloud.io/project/issues?id=1058-Sophia-Gavrila-Forge_Portfolio-Frontend&pullRequest=22&resolved=false&types=BUG) report which lists all the files where you should use `forEach()` instead of `map()` because we are not doing anything with the returned array that `map()` produces.

2. There are minor code smells like import duplications listed in [the SonarCLoud report here](https://sonarcloud.io/project/issues?id=1058-Sophia-Gavrila-Forge_Portfolio-Frontend&pullRequest=22&resolved=false&types=CODE_SMELL).  This can be resolved by separating multiple imports from the same place in one line like so: `import {url, portfolioUrl} from "../api/api";`

3. High percentage of code duplication as outlined in this [SonarCloud report here](https://sonarcloud.io/component_measures?id=1058-Sophia-Gavrila-Forge_Portfolio-Frontend&pullRequest=22&metric=new_duplicated_lines_density&view=list).  

Code duplication can be the most time instensive to resolve. 
First priority is to bring bugs to 0 and code smells to 0, then work on duplication

**Comment by sophiagavrila** [comment]
PR title should be more descriptive about the changes being brought into dev branch from this branch.  The description is good. 

You can go to the top of PR and click `Edit` to change PR title

**Comment by sophiagavrila** [comment]
2 bugs yet to be squashed: replace these 2 `map()`s with `forEach()`[ Here are the two reported bugs](https://sonarcloud.io/project/issues?id=1058-Sophia-Gavrila-Forge_Portfolio-Frontend&pullRequest=22&resolved=false&types=BUG)

**Review by sophiagavrila — APPROVED** [review]

**Review by ddinh — APPROVED** [review]

**Comment by sophiagavrila** [comment]
Awesome work and elimination of code smells/duplication issues.  Passes all quality gate checks and deployment tests. This branch is good to merge pending 1 more reviewer's approval.  Looped in @ddinh and @MadeFaux

**Pull request merged**
